C2CRIBuildDir/projects/C2C-RI/src/jameleon-test-suite-3_3-RC1-C2CRI/jameleon-core/src/java/net/sf/jameleon/ui/ConfigDialog.java

```diff
@@ -85,18 +84,12 @@ public void actionPerformed(ActionEvent evt) {
                         throw new IOException("Failed to create " + f.getAbsolutePath());
                       }
                   }
-                  os = new FileOutputStream(f);
-                  props.store(os, null);
+                  try (OutputStream os = new FileOutputStream(f))
+                  {
+                    props.store(os, null);
+                  }
               }catch(IOException ioe){
                   ioe.printStackTrace();
-              }finally{
-                  if (os != null) {
-                      try{
-                          os.close();
-                      }catch(IOException ioe){
-                          //So what, we couldn't close the file?
-                      }
-                  }
               }
               Configurator.clearInstance();
```

Comment:
Why makes the clearInstance() func static when this Configure instance is calling it ?

Reply:
The clearInstance() method on line 94 is a static method of the net.sf.jameleon.util.Configurator class. The clearInstance() methods I changed to be static were in the net.sf.jameleon.event package for the different Handler classes. Those methods needed to be changed to static since they only reference static member variables.

Reply:
Thanks!